### ISSUE TITLE
Default alerts aren't accessible.

### ISSUE THREAD

**Issue by mhemesath** [cerner/terra-core#1057]
# Issue Description
The default alert doesn't pass accessibility tests.

## Issue Type

- [ ] New Feature
- [ ] Enhancement
- [x] Bug
- [ ] Other

## Expected Behavior
Passes accessibility tests

## Current Behavior
> Element has insufficient color contrast of 3.52 (foreground color: #e50000, background color: #fed1d1, font size: 10.5pt, font weight: bold)

## Steps to Reproduce

1. Install [Axe Chrome Extension](https://chrome.google.com/webstore/detail/axe/lhdoppojpmngadmnindnejefpokejbdd?hl=en-US)
2. Navigate to the [alert example page](http://engineering.cerner.com/terra-core/#/tests/alert-tests/default)
3. Run axe by going to developer tools and clicking axe tab.

**Comment by PaulSolDev** [comment]
JIRA created

**Comment by neilpfeiffer** [comment]
# Design Input

Update `foreground color: #e50000` to **`#cc0000`**
Update `background color: #fed1d1` to **`#fedcdc`**

Keep same: `font size: 10.5pt, font weight: bold`
